tests/c/CMakeLists.txt

```diff
@@ -20,34 +20,22 @@ if(BUILD_TESTS)
     )
 
     target_include_directories(unittests PRIVATE
-        ${CMAKE_CURRENT_BINARY_DIR}/../../H
-        ${CMAKE_SOURCE_DIR}/interfaces)
-
-    # Required because we're compiling the tests written in C as C++
-    set(CMAKE_CXX_FLAGS "-fpermissive -Wwrite-strings -std=c++0x")
-
-    if(USE_VCPKG)
-        set_property(TARGET unittests PROPERTY CXX_STANDARD 14)
-
-        target_link_libraries(unittests PRIVATE
-            GTest::gtest_main ${CSOUNDLIB_STATIC} SndFile::sndfile)
-    else()
-        set(CMAKE_CXX_FLAGS "-fpermissive -Wwrite-strings -std=c++0x")
-
-        if (APPLE)
-            set(CMAKE_CXX_FLAGS "-stdlib=libc++")
-        endif()
-
-        target_include_directories(unittests PRIVATE
-            ${GTEST_INCLUDE_DIRS})
+        ${CMAKE_SOURCE_DIR}/H
+        ${CMAKE_SOURCE_DIR}/interfaces
+    )
+    
+    set_property(TARGET unittests PROPERTY CXX_STANDARD 14)
```

Comment:
Should be `target_link_options` maybe?

Reply:
Hmm, what do you mean?

Reply:
https://cliutils.gitlab.io/modern-cmake/chapters/features/cpp11.html should set the standard via the `target_compile_features` method I mean. Not the link_options sorry

Reply:
https://github.com/csound/csound/pull/1786/files#diff-7424ebd04f7f2f0b6a6f91f4a25c6fec73f2f5cc33d1b519db3cb4523dfd16c3R38 I've used this elsewhere and it works. In fact, maybe let me merge the changes I've proposed there and see what is applicable after